README.md

```diff
@@ -0,0 +1,21 @@
+# Build
```

Comment:
Hi!
I have already added readme to master but have not rebased this branch yet. Could you remove the readme commit and modify current readme instead?